.meshrc.yaml

```diff
@@ -161,7 +161,7 @@ sources:
             path: /bulk_download
             method: POST
             requestSchema: ./json-schemas/createBulkDownload.json
-            responseName: BulkDownload
+            responseTypeName: BulkDownload
```

Comment:
@rainandbare should this change be made?  It seems to fix the warning, but I wanted to make sure this is ok.

Reply:
Oh amazing! Thank you for finding that!